packages/@dcl/sdk/types/tsconfig.ecs7.json

```diff
@@ -22,23 +22,11 @@
       "ES2020"
     ],
     "types": [
-      "env",
-      "ecs7",
-      "rpc-modules/CommunicationsController",
-      "rpc-modules/DevTools",
-      "rpc-modules/EngineApi",
-      "rpc-modules/EnvironmentApi",
-      "rpc-modules/EthereumController",
-      "rpc-modules/ParcelIdentity",
-      "rpc-modules/Permissions",
-      "rpc-modules/Players",
-      "rpc-modules/PortableExperiences",
-      "rpc-modules/SignedFetch",
-      "rpc-modules/SocialController",
-      "rpc-modules/UserActionModule",
-      "rpc-modules/UserIdentity"
+      "@dcl/js-runtime",
+      "ecs7"
```

Comment:
what is ecs7?

Reply:
ecs7 is the lib to use the ecs7 (and the runtime initialization) 
`js-runtime` only has minimal exposure by the `scene-runtime`. You could use `@dcl/js-runtime` and compile some custom ecs or lib.

Reply:
OH perfect